client/additional-methods-setup/upe-preview-methods-selector/add-payment-methods-task.js

```diff
@@ -193,8 +192,7 @@ const AddPaymentMethodsTask = () => {
 				completeActivation( itemId );
 			}
 		} else {
-			paymentMethodsState[ itemId ] = false;
-			handlePaymentMethodChange( paymentMethodsState );
+			handlePaymentMethodChange( itemId, false );
```

Comment:
Same here.

Reply:
Thanks, I have updated it here - 55ec5d7